demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -2769,7 +2769,7 @@ export class DemoMeetingApp
       return;
     }
     const tileIndex = tileState.localTile
-      ? 16
+      ? 27
```

Comment:
Should this be 25 instead of 27? Checking since, the earlier number was just the total number of local video tiles.

Reply:
Another question:
In the [video integration test](https://github.com/aws/amazon-chime-sdk-js/blob/master/integration/js/checks/LocalVideoCheck.js#L25), when we are checking the local tile, we actually check the tile with index 17. How does it correlate with tileIndex 16 here?

Reply:
I don't think the previous number is correct. Content share also uses video tile and also got assigned one in this function so 27 sounds more correct to me.

Reply:
iirc, it is due to the difference in logic between the demo and integration test. For integration test, we do not have tileOrganizer and it basically just counts the number of video elements in the page (https://github.com/aws/amazon-chime-sdk-js/blob/master/integration/js/pages/AppPage.js#L389). 
We have a [`content-share-video` element ](https://github.com/aws/amazon-chime-sdk-js/blob/master/demos/browser/app/meetingV2/meetingV2.html#L527) which seems to be counted first, thus the index is shifted by one (so 16 to 17).